src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
-    // @checkstyle MagicNumberCheck (1 line)
-    @Test(expected = TimeoutException.class, timeout = 1000)
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting.
+    //  Needs to create a new one. I think this can looks like this:
+    //  new Wait().seconds(long seconds). Needs to do this in the test
```

Comment:
@aivinog1 I'm not sure this is needed and I don't really like the proposed solution. "Wait" is an action, not a noun, hence cannot be made a proper object.

Reply:
Good point. I've thought about it again and decide that TimeUnit not so bad decision.